modules/oe_webtools_page_feedback/tests/src/FunctionalJavascript/PageFeedbackFormTest.php

```diff
@@ -24,9 +28,23 @@ class PageFeedbackFormTest extends WebDriverTestBase {
   protected $defaultTheme = 'stark';
 
   /**
-   * Tests Page Feedback Form configuration.
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    $this->drupalPlaceBlock('oe_webtools_page_feedback_form');
+    $this->drupalPlaceBlock('page_title_block');
+    $this->drupalCreateContentType(['type' => 'page', 'name' => 'Basic page']);
+    ConfigurableLanguage::createFromLangcode('pt')->save();
```

Comment:
What language is `pt`? Because the Drupal language code for Portuguese is `pt-pt`.

Reply:
fixed